service/controller/resource/pvc/desired.go

```diff
@@ -20,15 +21,36 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 	if key.StorageType(customObject) == "persistentVolume" {
 		r.logger.Debugf(ctx, "computing the new PVCs")
 
-		PVCs, err = newEtcdPVCs(customObject)
+		etcdPVCs, err := newEtcdPVCs(customObject)
 		if err != nil {
 			return nil, microerror.Mask(err)
 		}
 
+		PVCs = append(PVCs, etcdPVCs...)
+
 		r.logger.Debugf(ctx, "computed the %d new PVCs", len(PVCs))
 	} else {
 		r.logger.Debugf(ctx, "not computing the new PVCs because storage type is not 'persistentVolume'")
 	}
 
+	if key.HasHostVolumes(customObject) {
+		r.logger.Debugf(ctx, "computing the new worker PVCs")
+
+		// Retrieve the existing Persistent Volume in the management cluster to get the storage size
+		// and create the Persistent Volume Claims for the workload cluster's workers
+		pvsList, err := r.k8sClient.CoreV1().PersistentVolumes().List(ctx, metav1.ListOptions{LabelSelector: LabelMountTag})
```

Comment:
I believe the label selector should be of the form `key=value`. `LabelMountTag ` is `mount-tag` which is missing the value part.

Reply:
I thought that it would work same as `kubectl`. if you run `kubectl get pv -lmount-tag` it works. so then it is required the `key=value`? My assumption is no.

Reply:
Oh right. I just checked the docs and this should work.